apps/posts/src/views/comments/comments.tsx

```diff
@@ -0,0 +1,138 @@
+            if (filter.values[0]) {
+                if (filter.operator === 'is_not') {
+                    parts.push(`post_id:-${filter.values[0]}`);
+                } else {
+                    // Default to 'is' operator
+                    parts.push(`post_id:${filter.values[0]}`);
+                }
+            }
+            break;
+        case 'author':
+            if (filter.values[0]) {
+                if (filter.operator === 'is_not') {
+                    parts.push(`member_id:-${filter.values[0]}`);
+                } else {
+                    // Default to 'is' operator
+                    parts.push(`member_id:${filter.values[0]}`);
+                }
+            }
+            break;
+        }
+    }
+    
+    return parts.length ? parts.join('+') : undefined;
+}
+
+const Comments: React.FC = () => {
+    const [filters, setFilters] = useState<Filter[]>([]);
+    
+    const apiFilter = useMemo(() => buildCommentsFilter(filters), [filters]);
+    
+    const {
+        data,
+        isError,
+        isLoading,
+        isFetchingNextPage,
+        fetchNextPage,
+        hasNextPage
+    } = useBrowseComments({
+        searchParams: apiFilter ? {filter: apiFilter} : {}
+    });
+
+    const handleAddFilter = useCallback((field: string, value: string, operator: string = 'is') => {
+        setFilters((prevFilters) => {
+            // Remove any existing filter for the same field
+            const filtered = prevFilters.filter(f => f.field !== field);
+            // Add the new filter
+            return [...filtered, createFilter(field, operator, [value])];
+        });
+    }, []);
+
+    return (
+        <CommentsLayout>
+            <CommentsHeader filters={filters} onFiltersChange={setFilters} />
+            <CommentsContent>
+                {isLoading ? (
+                    <div className="flex h-full items-center justify-center">
+                        <LoadingIndicator size="lg" />
+                    </div>
+                ) : isError ? (
+                    <div className="mb-16 flex h-full flex-col items-center justify-center">
+                        <h2 className="mb-2 text-xl font-medium">
+                            Error loading comments
+                        </h2>
+                        <p className="mb-4 text-muted-foreground">
+                            Please reload the page to try again
+                        </p>
+                        <Button onClick={() => window.location.reload()}>
+                            Reload page
+                        </Button>
+                    </div>
+                ) : !data?.comments.length ? (
+                    <div className="flex h-full items-center justify-center">
+                        <EmptyIndicator
+                            title="No comments yet"
+                        >
+                            <LucideIcon.MessageSquare />
+                        </EmptyIndicator>
+                    </div>
```

Comment:
Minor future enhancement: It would be nice to change the wording to "No comments match selected filter" or similar when a filter is in use